README.md

```diff
@@ -2,10 +2,27 @@
 
 This guide will take you through the steps of deploying Kubernetes to Openstack using docker. The primary concept is described here: [docker-multinode](https://github.com/FujitsuEnablingSoftwareTechnologyGmbH/kubernetes/blob/master/docs/getting-started-guides/docker-multinode/master.md)
```

Comment:
This guide will take you through the steps of deploying Kubernetes to bare metal or Openstack.

Remove the other part. it is not correct